src/main/java/com/r307/arbitrader/service/model/PaperAccountService.java

```diff
@@ -0,0 +1,53 @@
+package com.r307.arbitrader.service.model;
+
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.account.*;
+import org.knowm.xchange.exceptions.ExchangeException;
+import org.knowm.xchange.exceptions.NotAvailableFromExchangeException;
+import org.knowm.xchange.exceptions.NotYetImplementedForExchangeException;
+import org.knowm.xchange.instrument.Instrument;
+import org.knowm.xchange.service.account.AccountService;
+import org.knowm.xchange.service.trade.params.DefaultWithdrawFundsParams;
+import org.knowm.xchange.service.trade.params.TradeHistoryParams;
+import org.knowm.xchange.service.trade.params.WithdrawFundsParams;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+public class PaperAccountService implements AccountService {
+
+    AccountService accountService;
+
+    Currency homeCurrency;
+
+    public PaperAccountService (AccountService accountService, Currency homeCurrency) {
+        this.accountService=accountService;
+        this.homeCurrency=homeCurrency;
+    }
+
+    public AccountInfo getAccountInfo() {
+        List<Wallet> wallets = new ArrayList<>();
+        List<Balance> balances = new ArrayList<>();
+        balances.add(
+            new Balance(
+                this.homeCurrency,
+                new BigDecimal(100),
+                new BigDecimal(100),
+                new BigDecimal(0)));
+        wallets.add(Wallet.Builder.from(balances).id(UUID.randomUUID().toString()).build());
+        return new AccountInfo (wallets);
```

Comment:
Not necessary to fix now, but this will make some of the log messages look weird if the balance is always 100.

Reply:
The latest commit updates the paper account balance after filling an order.